feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/component/AboutCredits.kt

```diff
@@ -1,45 +1,69 @@
 package io.github.droidkaigi.confsched2023.about.component
 
-import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.lazy.LazyListScope
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import io.github.droidkaigi.confsched2023.about.AboutStrings
+import io.github.droidkaigi.confsched2023.feature.about.R
 
-const val AboutCreditsSponsorsItemTestTag = "AboutCreditsSponsorsItem"
-const val AboutCreditsContributorsItemTestTag = "AboutCreditsContributorsItem"
 const val AboutCreditsStaffItemTestTag = "AboutCreditsStaffItem"
+const val AboutCreditsContributorsItemTestTag = "AboutCreditsContributorsItem"
+const val AboutCreditsSponsorsItemTestTag = "AboutCreditsSponsorsItem"
 
 fun LazyListScope.aboutCredits(
     onStaffItemClick: () -> Unit,
     onContributorsItemClick: () -> Unit,
     onSponsorsItemClick: () -> Unit,
 ) {
     item {
-        Text("Credits")
+        Text(
+            text = AboutStrings.CreditsTitle.asString(),
+            style = MaterialTheme.typography.titleMedium,
+            modifier = Modifier
+                .padding(
+                    start = 16.dp,
+                    top = 32.dp,
+                    end = 16.dp,
+                ),
+        )
     }
     item {
-        Text(
+        AboutContentColumn(
```

Comment:
Now that we can't see the AboutContentColumn for Screenshot Tests, could you provide a scrolled screenshot test?  📸